moved/src/block/gas.rs

```diff
@@ -101,26 +101,31 @@ impl GasFee for Eip1559GasFee {
     }
 }
 
-#[cfg(test)]
-mod tests {
+#[cfg(any(feature = "test-doubles", test))]
```

Comment:
What do we mean by `test-doubles` here? Test duplicate or a second test?

Reply:
Inspired by https://martinfowler.com/bliki/TestDouble.html

Reply:
After talking about the purpose of test doubles, realized that test only code doesn't carry between crates, so it's needed for the tests from external crates.